docs/getting_started.rst

```diff
@@ -37,7 +37,7 @@ Event consumers may never want to lose certain events even after a brief failure
 Supported events and mapping of edx events onto xAPI and Caliper formats
 ------------------------------------------------------------------------
 
-List of supported edx events can be found in `Supported_events <./event-mapping/Supported_events.rst>`_ along with their mapping onto xAPI and Caliper format.
+List of supported edx events can be found in `Supported_events <https://github.com/openedx/event-routing-backends/blob/master/docs/event-mapping/Supported_events.rst>`_ along with their mapping onto xAPI and Caliper format.
```

Comment:
```suggestion
List of supported edx events can be found in :doc:`Supported Events <event-mapping/Supported_events>` along with their mapping onto xAPI and Caliper format.
```

We shouldn't link off site for documents that are in this repo, we should be able reference them directly like this.

Reply:
we explicitly link off site here because Supported events page on github links to a lot of other json documents which are on github. If we don't link off site here we'll have to change all those links pointing to github in the Supported events page.

Reply:
Hmm, I think the correct thing to do would be to correct all the github RST links to be relative links to stay in the sphinx docs as much as possible except for the actual samples.  Having some rendered RST and some raw RST seems like a weird mix.

Reply:
Yeah all RST links are relative link except this link in `getting_started.rst` and reason for that is it needs to point to github.com version of `Supported_events.rst` instead of sphinx version. Most of the links in `Supported_events.rst` are relative and point to samples which reside on github.com too. If we make this link relative here and point to sphinx version of `Supported_events.rst` then we'll have to make all samples links absolute since they are on github.com so I think keep a single link absolute(offsite) here is better than option. One more thing raw RST of `Supported_events.rst` looks better than rendered RST.